tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/im2latex_evaluator.py

```diff
@@ -0,0 +1,306 @@
+            launcher_config.get('tags'),
+            dataset_config['name']
+        )
+
+
+class BaseModel:
+    def __init__(self, network_info, launcher, default_model_suffix):
+        self.default_model_suffix = default_model_suffix
+        self.network_info = network_info
+
+    def predict(self, idenitifers, input_data):
+        raise NotImplementedError
+
+    def release(self):
+        pass
+
+    def automatic_model_search(self, network_info):
+        model = Path(network_info['model'])
+        if model.is_dir():
+            is_blob = network_info.get('_model_is_blob')
+            if is_blob:
+                model_list = list(model.glob('*{}.blob'.format(self.default_model_suffix)))
+                if not model_list:
+                    model_list = list(model.glob('*.blob'))
+            else:
+                model_list = list(model.glob('*{}.xml'.format(self.default_model_suffix)))
+                blob_list = list(model.glob('*{}.blob'.format(self.default_model_suffix)))
+                if not model_list and not blob_list:
+                    model_list = list(model.glob('*.xml'.format(self.default_model_suffix)))
+                    blob_list = list(model.glob('*.blob'.format(self.default_model_suffix)))
+                    if not model_list:
+                        model_list = blob_list
+            if not model_list:
+                raise ConfigError('Suitable model for {} not found'.format(self.default_model_suffix))
+            if len(model_list) > 1:
+                raise ConfigError('Several suitable models for {} found'.format(self.default_model_suffix))
+            model = model_list[0]
+        if model.suffix == '.blob':
+            return model, None
+        weights = network_info.get('weights', model.parent / model.name.replace('xml', 'bin'))
+
+        return model, weights
+
+
+def create_recognizer(model_config, launcher, suffix):
+    """Creates encoder and decoder element for Im2LaTeX model
+
+    Args:
+        model_config ([type]): [description]
+        launcher ([type]): [description]
+        suffix ([type]): [description]
+
+    Raises:
+        ValueError: [description]
+
+    Returns:
+        [type]: [description]
+    """
+    launcher_model_mapping = {
+        'dlsdk': RecognizerDLSDKModel
+    }
+    framework = launcher.config['framework']
+    model_class = launcher_model_mapping.get(framework)
+    if not model_class:
+        raise ValueError('model for framework {} is not supported'.format(framework))
+    return model_class(model_config, launcher, suffix)
+
+
+class SequentialModel:
+    def __init__(self, network_info, launcher, models_args, meta, is_blob=None):
+        recognizer_encoder = network_info.get('recognizer_encoder', {})
+        recognizer_decoder = network_info.get('recognizer_decoder', {})
+        if 'model' not in recognizer_encoder:
+            recognizer_encoder['model'] = models_args[1 if len(models_args) > 1 else 0]
```

Comment:
do you have 3 parts in your model or why indexing started with 1?
usually, model args is the list with several paths if you provide in command line
-m <model1> <model2>

Reply:
Fixed this